src/mumble/AudioInput.cpp

```diff
@@ -1048,6 +1048,11 @@ void AudioInput::encodeAudioFrame(AudioChunk chunk) {
 				if (!qetLastMuteCue.isValid() || qetLastMuteCue.elapsed() > MUTE_CUE_DELAY) {
 					qetLastMuteCue.start();
 					ao->playSample(Global::get().s.qsTxMuteCue, Global::get().s.cueVolume);
+
+					if (!Global::get().s.bMuteCueShown && !Global::get().bInConfigUI) {
```

Comment:
I would suggest always emitting the signal ``doMuteCue`` and moving the "only show popup when" logic into the MainWindow slot.